File: 2292-counting-words-with-a-given-prefix/counting-words-with-a-given-prefix.cpp
class Solution {
public:
    int prefixCount(vector<string>& words, string pref) {
        int res = 0;
        int n = pref.size();
        bool flag = false;
        for(string s: words){
            for(int i=0;i<n;i++){
                if(s[i]==pref[i]){
                    flag = true;
                }
                else{
                    flag = false;
                    break;
                }
            }
            if(flag==true) res++;
        }
        return res;
    }
};

File: 2292-counting-words-with-a-given-prefix/counting-words-with-a-given-prefix.java
class Solution {
    public int prefixCount(String[] words, String pref) {
        int res = 0;
        int n = pref.length();
        Boolean flag = false;
        for(String s:words){
            if(s.startsWith(pref)) res++;
        }
        return res;
    }
}